packages/kit/src/views/Swap/pages/components/PreSwapDialogContent.tsx

```
@@ -18,6 +19,7 @@ import {
   ESwapTxHistoryStatus,
 } from '@onekeyhq/shared/types/swap/types';
 
+import { useActiveAccount } from '../../../../states/jotai/contexts/accountSelector';
```

ESLint

[error] Too many parent references (4 while maximum of 3 allowed)

(import-path/parent-depth)